src/getHumanAge.test.js

```diff
@@ -1,10 +1,45 @@
 'use strict';
 
-describe('getHumanAge', () => {
+describe(`getHumanAge`, () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0's if both inputs <15`, () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 1 if inputs are 15-24`, () => {
+    expect(getHumanAge(17, 19))
+      .toEqual([1, 1]);
+  });
+
+  test(`should return 2 if inputs are 24-28`, () => {
```

Comment:
You need to write "24-27" instead of "24-28" because 28 cat's ages would be 3 human ages and 28 dog's ages would be 2 human ages.